recipes/cryptopp/all/conanfile.py

```diff
@@ -90,7 +105,6 @@ def _configure_cmake(self):
         return self._cmake
 
     def build(self):
-        self._patch_sources()
```

Comment:
no, keep the patch in build() please

Reply:
No problem with that but I am curious why.
I did that because otherwise this development workflow does not work:
```shell
conan install
conan source
conan build
...
conan build
```
Second conan build fail because it tries to patch the sources that are already patched.
The alternative is to call conan create each time, but building from scratch when developping/debugging the package is annoying.

Reply:
@fdgStilla dev workflow is incompatible with CCI, use `conan create` instead. Keep source folder/step untouched, only the downloaded file, no patches, no changes. Otherwise, for any change during the development, Conan will download all stuff again.